src/app/cube/content-pages/contribute-page/contribute-page.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from '@angular/core';
+import { sections } from './copy';
+
+@Component({
+  selector: 'clark-contribute-page',
+  templateUrl: './contribute-page.component.html',
+  styleUrls: ['./contribute-page.component.scss']
+})
+
+export class ContributePageComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void { }
+
+  title = 'Contribute';
+  copy = sections;
```

Comment:
If you remove copy like @mfranz1 suggested then delete this line too.
```suggestion
```

Reply:
Did it to both and tested on local environment to make sure it still functioned (and it did).